stylesheets/_component.tables.scss

```diff
@@ -17,33 +17,33 @@
         }
     }
 
-    [href],
-    .continuum-cms & [href]:link {
-        border-bottom: 1px dotted color(grey, light);
-        color: color(link);
-        text-decoration: none;
-
-        &:not(.btn):focus {
-            @include pulsar-link-focused;
-        }
-
-        &:focus {
-            border: 0;
-            color: color(black);
-        }
-    }
-
-    [href].btn {
-        border-bottom: 0;
-    }
+    // [href],
```

Comment:
Assume we're looking to replace these styles with a class?

Reply:
Maaaybe, it's one to discuss for sure